levelfile.txt

```diff
@@ -0,0 +1,42 @@
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
Wäre ganz schön, wenn Pull Requests immer nur zu einem Thema sind. Also für die Levels einen eigenen auf machen. Kann ich jetzt aber auch erstmal so machen. Allerdings sollte der levelfile, in den Ordner `resources` gelegt werden

Reply:
Macht ihr das noch noch? Dann kann ich mergen